rllib/utils/exploration/exploration.py

```diff
@@ -66,21 +65,21 @@ def get_exploration_action(self,
         pass
 
     def get_loss_exploration_term(self,
-                                  model_output,
-                                  model=None,
-                                  action_dist=None,
-                                  action_sample=None):
+                                  model_output: TensorType,
```

Comment:
Should we remove this unused method? Not sure if it's needed.

Reply:
For now, yes. We won't need it for a while. Want to do param noise next.